ggml/src/ggml-backend.cpp

```diff
@@ -1649,10 +1652,12 @@ void ggml_backend_view_init(struct ggml_tensor * tensor) {
 
     tensor->buffer = tensor->view_src->buffer;
     tensor->data = (char *)tensor->view_src->data + tensor->view_offs;
-    ggml_backend_buffer_init_tensor(tensor->buffer, tensor);
+    ggml_status status = ggml_backend_buffer_init_tensor(tensor->buffer, tensor);
+    if (status != GGML_STATUS_SUCCESS)
+        GGML_LOG_WARN("init tensor failed: status = %d\n", status);
```

Comment:
Don't just print a meaningless error code. There is already a `ggml_status_to_string` function. Remember to format the code with the same style, or use `clang-format` if unsure (missing braces in the code block).

Reply:
- did nt know about ggml_status_to_string(). Retouched.
- was running clang-format but using clangtools14, so too old to enforce "insertBraces". Upgrading to clangtools 15+.